src/Tizen.NUI.Components/Controls/SelectButton.cs

```diff
@@ -173,21 +190,26 @@ protected override bool HandleControlStateOnTouch(Touch touch)
                 return false;
             }
 
-            PointStateType state = touch.GetState(0);
-            bool ret = base.HandleControlStateOnTouch(touch);
-            switch (state)
+            if (IsSelected == true)
             {
-                case PointStateType.Up:
-                    OnSelect();
-                    break;
-                default:
-                    break;
+                isPressedWhileSelected = true;
             }
+
+            var ret = base.HandleControlStateOnTouch(touch);
+
+            if ((EnableToggleSelected == false) && (isPressedWhileSelected == true))
+            {
+                IsSelected = true;
+                isPressedWhileSelected = false;
+
+                return false;
```

Comment:
The return value indicates whether this method call consumed touch or not.
The button is returning true in `HandleControlStateOnTouch` unless it's disabled.
So for this case, since it's not disabled and has reacted to touch with GUI changes so I think it would be nice to return the same value as what base.HandleControlStateOnTouch() has returned.